index.bs

```diff
@@ -3881,6 +3965,32 @@ To <dfn>check if an [=attribution source=] can create [=aggregatable contributio
     |remainingAggregatableBudget|, return false.
 1. Return true.
 
+To <dfn>find matching aggregatable bucket</dfn> given an [=attribution trigger=] |trigger| and an [=attribution source=] |sourceToAttribute|:
+1. [=list/iterate|For each=] [=aggregatable bucket=] |aggregatableBucket| of |trigger|'s [=attribution trigger/aggregatable buckets=]:
+    1. If the result of running [=match an attribution source against filters and negated filters=]
+        with |sourceToAttribute|, |aggregatableBucket|'s [=aggregatable bucket/filters=],
+        |aggregatableBucket|'s [=aggregatable bucket/negated filters=], and
+        |trigger|'s [=attribution trigger/trigger time=] is true:
+        1. Return |aggregatableBucket|'s [=aggregatable bucket/bucket=].
+1. Return null.
+
+To <dfn>get remaining aggregatable budget for matched bucket</dfn> given an [=attribution source=] |sourceToAttribute|,
```

Comment:
This is only invoked once, and can just be inlined.

Reply:
Nice catch. Done.